dist/ccdb5.css

```diff
@@ -1,3 +1,3 @@
ody{font-size:1.125em;font-weight:400;letter-spacing:inherit;line-height:1.25;text-transform:inherit}}.m-pull-quote__citation{color:var(--gray);font-size:.875em;font-weight:600;letter-spacing:1px;line-height:1.25;text-transform:uppercase;margin-bottom:1.0714285714em}p+.m-pull-quote__citation,ul+.m-pull-quote__citation,ol+.m-pull-quote__citation,dl+.m-pull-quote__citation,figure+.m-pull-quote__citation,img+.m-pull-quote__citation,table+.m-pull-quote__citation,blockquote+.m-pull-quote__citation,h1+.m-pull-quote__citation,.h1+.m-pull-quote__citation,h2+.m-pull-quote__citation,.h2+.m-pull-quote__citation,h3+.m-pull-quote__citation,.h3+.m-pull-quote__citation,h4+.m-pull-quote__citation,.h4+.m-pull-quote__citation,h6+.m-pull-quote__citation,.h6+.m-pull-quote__citation{margin-top:2.1428571429em}.m-pull-quote__citation:before{content:"\2014"}.m-slug-header{border-top:1px solid var(--gray-40)}.m-slug-header__heading{display:inline-block;padding-top:.2857142857em;border-top:5px solid var(--green);margin-top:-3px;font-size:.875em;font-weight:600;letter-spacing:1px;line-height:1.25;text-transform:uppercase;margin-bottom:1.0714285714em}.a-tagline{font-size:.75rem;display:grid;grid-template-columns:22px 1fr;grid-column-gap:10px}.a-tagline__text{display:inline-block}.a-tagline .u-usa-flag{margin-top:1px}.a-tagline--large{font-size:1rem}.a-tagline--large .u-usa-flag{margin-top:4px}.u-usa-flag{display:inline-block;width:24px;height:13px;background-image:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAZCAMAAABAf11LAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAE5QTFRF////sxlC7MbQ2YyhxlNxCjFhR2WJV3GSKUt1dYumOFh/GT5rhZiwwszYsr/OlKW6Zn6c0djh8PL1iR9Ko7LE4OXrl0pttKC0pXWRtYKbSuJhRQAAANFJREFUeNrkkctuwyAUREnSuW/ApHYf//+jBVdZVcJi3aORgAXcMyLBAAJEzsVG3m8TkifyI3zfPQ6nJJLo421CArSBmkgjNEWtQE4zXJmClXuCWIlU5hdQxCqbqnE1KdIz79CVDvBwZxyKfQfmHTyzl01UZSvOWSTbhZLSWeDMufWLC/1ls3amT4qQq394EjIjApxBT+/nr8eEBNuKcB9SWMpmEXalNOylmlUZNTr4vE/4VdKhpC+leQf6y/e0wzL3RdJtkfUJyzwW+ZcdfgQYAJmJD3zerW6OAAAAAElFTkSuQmCC);background-size:contain;background-repeat:no-repeat}.introjs-tooltip.wide{max-width:80vw!important;width:80vw;margin-left:-40vw}.introjs-tooltip.wide .left{width:80%;float:left}.introjs-tooltip.wide .right{display:inline-block;margin-left:15px}@media screen and (min-width: 1800px){.introjs-tooltipReferenceLayer>.first-step{margin-top:-40vh!important}}@media screen and (min-width: 901px)and (max-width: 1799px){.introjs-tooltipReferenceLayer>.first-step{margin-top:-90vh!important}}@media screen and (min-width: 750px)and (max-width: 900px){.introjs-tooltipReferenceLayer>.first-step{margin-top:-110vh!important}}@media screen and (min-width: 600px)and (max-width: 749px){.introjs-tooltipReferenceLayer>.first-step{margin-top:-130vh!important}}@media screen and (max-width: 599px){.introjs-tooltipReferenceLayer>.first-step{margin-top:-110vh!important}}.tour-button{background-color:var(--pacific);height:40px;color:var(--white);border-radius:5px;position:fixed;bottom:21px;right:25px}
```

Comment:
No changes needed to the `dist` folder, but is its content something that we really have to track or even push to 
 our repo? It can definitely be ignored via a `.gitignore` file, but I'm not sure of the actual reason we track it.

Reply:
Yeah, it's part of the build and deploy process.  

It's how cfgov includes the assets.  All of the libraries are also in an offline cache too.